ROADMAP.md

```diff
@@ -0,0 +1,98 @@
+Ansible Roadmap
+=============
```

Comment:
At some point, and confirming with Marketing isn't a bad thing here, we'll need to introduce the "Ansible by Red Hat" as a product title. Seeing the doc title here made me think of it.  Should be added in the initial README as well as other places where more formal branding is needed (general content, like everything below and this title as well, can remain as Ansible).  Sorry, probably not the best place to make this comment, but worth noting.